docs/reference/objects/product/index.md

```diff
@@ -188,3 +214,14 @@ Returns the products [venue]({% link docs/reference/objects/product/venue.md %})
 
 Return the product [whats_included]({% link docs/reference/objects/product/whats_included.md %})
 
+
+<!--
+
+Methods not publicly documents:
```

Comment:
[Question] If these aren't publicly documented why are we listing them here?
[Typo] s/documents/documented

Reply:
Good question! Basically I wanted a way to track which methods had not been documented yet, either because they would link to drops that aren't documented yet or we have _chosen_ to not document them.

I wanted to add this comment just for developers, i.e. not an HTML comment. However, there doesn't seem to be a clean way to do that in markdown. I thought it was safe enough to have this hidden in the HTML.

Reply:
Looking at the ones I've listed, I remember I had intended to come back to these. So thanks for the reminder. I will take another look at these methods.